Pod/Classes/ADClusterAnnotation.m

```diff
@@ -8,8 +8,13 @@
 
 #import "ADClusterAnnotation.h"
 #import "TSRefreshedAnnotationView.h"
+#import "TSPlatformCompatibility.h"
 
+#if TS_TARGET_IOS
 #define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v) ([[[UIDevice currentDevice] systemVersion] compare:(v) options:NSNumericSearch] != NSOrderedAscending)
+#else
+#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v) YES
```

Comment:
This macro is currently only being used to check for a particular bug fix in iOS 8. The Mac support is intended to be 10.10+, and none of those systems have that particular bug, so we just say "YES".